### PR TITLE
Folders added: Javascrit assignment 10,12,13,14.

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

**Review by shubhamdalal49 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: vijaya/JavaScript/assignment10/Q1.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            regexp = /^[A-Z]/;
+            if (regexp.test(str)) {
+                document.getElementById("answer").value = "Uppercase";
+            } 
+            else {
+                document.getElementById("answer").value = "is not Uppercase";   
+            }
+        }
+    </script>
+    </head>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q1.html:43]: make correct indentation for head tag

File: vijaya/JavaScript/assignment10/Q8.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            height:200px;
+            margin: auto;
+            padding-top: 5px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .res{
+            background-color: white;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q8.html:30]: make correct indentation for head tag

File: vijaya/JavaScript/assignment10/Q7.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            height:200px;
+            margin: auto;
+            padding-top: 5px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .res{
+            background-color: white;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q7.html:30]: make correct indentation for style tag

File: vijaya/JavaScript/assignment10/Q6.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            height:250px;
+            margin: auto;
+            padding-top: 20px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .res{
+            background-color: white;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q6.html:32]: make correct indentation for style tag

File: vijaya/JavaScript/assignment10/Q6.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            str = document.getElementById("string").value;
+            regExp = /^(https:\/\/www\.|http:\/\/www\.|www\.)[a-zA-Z0-9\-_$]+\.[a-zA-Z]{2,3}$/;
+            if (regExp.test(str)) {
+                document.getElementById("Result").innerHTML = "This is valid URL";
+            } 
+            else {
+                document.getElementById("Result").innerHTML = "This is URL";
+            }
+        }
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q6.html:45]: make correct indentation for script tag

File: vijaya/JavaScript/assignment10/Q5.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            padding-bottom: 5px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .resultText {  
+            display: flex;
+            gap: 10px;
+            background-color: white;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q5.html:32]: make correct indentation for style tag

File: vijaya/JavaScript/assignment10/Q3.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+            height:200px;
+            margin: auto;
+            padding-top: 5px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .res {
+            background-color: white;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q3.html:30]: make correct indentation for style tag

File: vijaya/JavaScript/assignment10/Q2.html [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+            height:250px;
+            margin: auto;
+            padding-top: 5px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .result{
+            background-color: white;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment10/Q2.html:26]: make correct indentation for style tag

File: vijaya/JavaScript/assignment13/Q1.html [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+        let minutes = today.getMinutes();
+        let local = hours>=12?"PM":"AM";
+        hours = hours % 12;
+        if(minutes < 10) {
+            minutes = "0" + minutes
+        }
+        alert(today.toDateString()+",   " +date  +" : "+time+" "+local);    
+        document.getElementById("num1").innerHTML="Date is: "+date;
+        document.getElementById("num2").innerHTML="Time is: "+time +" "+local;
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment13/Q1.html:60]: make correct indentation for script tag

File: vijaya/JavaScript/assignment14/Q1.html [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+            .res{
+                margin-top: 30px;
+                padding-left: 30px;
+                padding-right: 30px;
+                background-color: white;
+                font-weight: bold;
+                font-size: 17px;
+                color:rgb(9, 46, 148);
+            }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment14/Q1.html:40]: head tag closing missing

File: vijaya/JavaScript/assignment14/Q2.html [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+            tr, td {
+                font-size: 14px;
+                min-width: 100px;
+                text-align: center;
+                padding-top: 5px;
+                padding-bottom: 3px;
+                border: 1px solid black;
+            }         
+        </style>
+    <body>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment14/Q2.html:55]: head tag closing missing

File: vijaya/JavaScript/assignment5/fibb.html [hunk 12]
```diff
@@ -4,12 +4,12 @@
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Pattern Demo</title>
+        <title>Fibbonacci Demo</title>
         <style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment5/fibb.html:8]: make correct indentation for style tag

File: vijaya/JavaScript/assignment7/ques3.html [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+                        array[j] = array[j + 1];
+                        array[j + 1] = temp;
+                    }
+                }
+            }
+            for (let k = 1; k <= size; k++) {
+                document.getElementById("Result1").innerHTML += "Marks " + y++ + " = " + array[k] + "<br>";
+            }
+        }
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques3.html:82]: make correct indentation for script tag

File: vijaya/JavaScript/assignment7/ques4.html [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+                        }
+                    }
+                }
+                for (let k = 1; k <= size; k++) {
+                    document.getElementById("Result1").innerHTML += "Marks " + y++ + " = " + array[k] + "<br>";
+                }
+            }
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques4.html:90]: remove extra lines

File: vijaya/JavaScript/assignment7/ques5.html [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+            width: 150px;
+            padding-left: 10px;
+        }
+        .res1 {
+            background-color: rgb(243, 233, 96);
+            width: 150px;
+            padding-left: 10px;
+        }
+    </style>
+    </head>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques5.html:39]: make correct indentation for head tag

File: vijaya/JavaScript/assignment7/ques5.html [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+                    array[j+1] = array[j];
+                    j--;
+                    array[j+1] = temp;
+                }
+            }
+            for (let k = 1; k <= size; k++) {
+                document.getElementById("Result1").innerHTML += "Marks " + y++ + " = " + array[k] + "<br>";
+            }
+        }
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques5.html:87]: make correct indentation for script tag

File: vijaya/JavaScript/assignment7/triangle.html [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+            background-color: white;
+            color: rgb(46, 49, 245);
+            width: 110px;
+            margin-left: 220px;
+        }
+        .btn{
+            padding: 6px;
+            text-align: center;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/triangle.html:29]: make correct indentation for style tag

File: vijaya/JavaScript/assignment7/triangle.html [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+                        document.getElementById("answer").innerHTML += " "+number;  
+                    }
+                    else{
+                        document.getElementById("answer").innerHTML += " "+j;
+                    }
+                } 
+                document.getElementById("answer").innerHTML +="<br>";
+            }
+        }
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/triangle.html:48]: make correct indentation for script tag

File: vijaya/JavaScript/assignment7/triangle.html [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+    </head>
+        <body style="background-color: rgb(102, 101, 101);">
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/triangle.html:50]: make correct indentation for body tag

File: vijaya/JavaScript/assignment7/triangle.html [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+                <br>
+                Enter the number of row: <input id="row"></br></br>
+                <div class="one">
+                <p><span id="answer"></span></p>
+                </div> 
+                <div class="btn">
+                <button onclick="floyds()" style=" background-color: rgb(17, 175, 223); height: 30px">Show</button></br></br>
+                </div>      
+            </div>
+        </body>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/triangle.html:63]: make correct indentation for body tag

File: vijaya/JavaScript/assignment8/Que1.html [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+            padding-top: 25px;
+            gap: 10px;
+            border-radius: 10px;
+        }
+        .res{
+            background-color: white;
+            width: 200px;
+            padding-left: 10px;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que1.html:33]: make correct indentation for style tag

File: vijaya/JavaScript/assignment8/Que1.html [hunk 22]
```diff
@@ -0,0 +1,70 @@
...
+        }
+        function display() {
+            var e = "",total=0;
+            for (var y = 1; y <=size; y++) {
+                total=(array[y]*20/100)+array[y];
+                e += "subject: " + y + " = " +total+ "<br/>";
+            }
+            document.getElementById("Result").innerHTML = "Total score of students:<br>"+e;
+        }
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que1.html:68]: make correct indentation for script tag

File: vijaya/JavaScript/assignment8/Que2.html [hunk 23]
```diff
@@ -0,0 +1,100 @@
...
+            padding-top: 25px;
+            gap: 5px;
+            border-radius: 10px;
+        }
+        .res {
+            background-color: white;
+            width: 200px;
+            padding-left: 10px;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que2.html:33]: make correct indentation for style tag

File: vijaya/JavaScript/assignment8/Que2.html [hunk 24]
```diff
@@ -0,0 +1,100 @@
...
+            return i + 1;
+        }
+        function quickSort(array, low, high) {
+            if (low < high) {
+                pi = partition(array, low, high);
+                quickSort(array, low, pi - 1);
+                quickSort(array, pi + 1, high);
+            }
+        }
+        </script>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que2.html:98]: make correct indentation for script tag

File: vijaya/JavaScript/assignment8/Que3.html [hunk 25]
```diff
@@ -0,0 +1,173 @@
...
+            gap: 5px;
+            border-radius: 10px;
+        }
+        .res{
+            background-color: white;
+            width: 200px;
+            padding-left: 10px;
+            margin-left: 140px;
+        }
+        </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que3.html:32]: make correct indentation for style tag

File: vijaya/JavaScript/assignment8/Que3.html [hunk 26]
```diff
@@ -0,0 +1,173 @@
...
+            array[i + 1] = temp;
+            return i + 1;
+        }
+        function quickSort(array, low, high) {
+            if (low < high) {
+                pi = partition(array, low, high);
+                quickSort(array, low, pi - 1);
+                quickSort(array, pi + 1, high);
+            }
+        }
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que3.html:170]: make correct indentation for script tag

File: vijaya/JavaScript/assignment8/Que4.html [hunk 27]
```diff
@@ -0,0 +1,215 @@
...
+                        document.getElementById("result").innerHTML += `Sorted Marks ${n++} &nbsp;: &ensp; <span style="font-size:18px;">${parseInt(merge[i])}</span> ` + "<br>";
+                    } else if (i > 8 && i < 10) {
+                        document.getElementById("result").innerHTML += `Sorted Marks ${n++} &nbsp;:&nbsp; <span style="font-size:18px;">${parseInt(merge[9])}</span> ` + "<br>";
+                    }
+                }
+                document.getElementById("after_sorting").innerHTML += merge.join(" &nbsp; ") + "<br>";
+            }
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment8/Que4.html:215]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make changes asap

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**